scripts/create-client-config.sh

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env bash
+#
+# k8ssandra-operator should be installed in the remote clusters prior to running this. The
+# script fetches the k8ssandra-operator service account from the remote cluster and
+# extracts the token and CA cert which are then added to a kubeconfig file. The script then
+# creates a secret with the contents of the kubeconfig file. Lastly, the script creates a
+# ClientConfig object that references the secret.
+#
+# This script requires the following to be installed:
+#
+#    - kubectl
+#    - yq
+#
+# TODO Accept multiple values for the src-context option and generate a kubeconfig with
+#      entries for each
+
+set -e
+
+OPTS=$(getopt -o h --long src-context:,src-kubeconfig:,dest-context:,dest-kubeconfig:,namespace:,serviceaccount,output-dir:,help -n 'create-client-config' -- "$@")
+
+eval set -- "$OPTS"
+
+function help() {
+    echo
+    echo "Syntax: create-client-config.sh [options]"
+    echo "Options:"
+    echo "src-context     The context for the source cluster that contains the service account. This or the src-kubeconfig option must be set."
+    echo "src-kubeconfig   The kubeconfig for the source cluster that contains the service account. This or the src-context option must be set."
+    echo "dest-context    The context for the cluster where the ClientConfig will be created."
+    echo "dest-kubeconfig  The kubeconfig for the cluster where the ClientConfig will be created."
+    echo "namespace       The namespace in which the service account exists and where the ClientConfig will be created."
+    echo "serviceaccount  The name of the service account from which the ClientConfig will be created. Defaults to k8ssandra-operator."
+    echo "output-dir      The directory where generated artifacts are written. If not specified a temp directory is created."
+}
+
+src_context=""
+src_kubeconfig=""
+dest_context=""
+dest_kubeconfig=""
+service_account="k8ssandra-operator"
+namespace=""
+output_dir=""
+
+while true; do
+  case "$1" in
+    --src-context ) src_context="$2"; shift 2 ;;
+    --src_kubeconfig ) src_kubeconfig="$2"; shift 2 ;;
+    --dest-context ) dest_context="$2"; shift 2 ;;
```

Comment:
I don't see `dest_context` or `dest_kubeconfig` being used anywhere below.

Reply:
I updated the script.